Refactor(eos_designs)!: Default number of OSPF next-hops in an ECMP route set to 128 (#5908)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

      ipv4_acl: IPV4
```

> ### `underlay_ospf_maximum_paths`: Default number of OSPF next-hops in an ECMP route set to 128

Reviewer: ```suggestion
### Default value of OSPF next-hops in an ECMP route `underlay_ospf_maximum_paths` is set to 128
```

> ### Default value of OSPF next-hops in an ECMP route `underlay_ospf_maximum_paths` is set to 128

Reviewer 2: ```suggestion
### Default value of `underlay_ospf_maximum_paths` is set to 128
```

In AVD 5.x, the parameter `underlay_ospf_maximum_paths`(maximum number of OSPF next-hops in an ECMP route) had no default value.

> Starting with AVD 6.0.0, this parameter now defaults to 128.

Reviewer: ```suggestion
Starting AVD 6.0.0, default value of `underlay_ospf_maximum_paths` is set to 128.
```

> Starting AVD 6.0.0, default value of `underlay_ospf_maximum_paths` is set to 128.

Reviewer 2: ```suggestion
Starting AVD 6.0.0, default value of `underlay_ospf_maximum_paths` is set to 128 to match with the EOS default value.
```

      ipv4_acl: IPV4
```

> ### Default value of `underlay_ospf_maximum_paths` set to 128

Reviewer 2: ```suggestion
### Default value of `underlay_ospf_maximum_paths` is set to 128
```

In AVD 5.x, the parameter `underlay_ospf_maximum_paths`(maximum number of OSPF next-hops in an ECMP route) had no default value.

Starting AVD 6.0.0, default value of `underlay_ospf_maximum_paths` is set to `128`.

> Since `128` is already the EOS system default, it will not be rendered in either the structured configuration or the CLI configuration.

Reviewer 3: ```suggestion
Since `128` is already the EOS system default, it will neither be rendered in the structured configuration nor the CLI configuration.
```
